.github/ISSUE_TEMPLATE/enhancement.yml

```
@@ -0,0 +1,17 @@
+name: Enhancement
+description: Suggest a new feature or improvement for the website.
+labels: [enhancement]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking your time to create your feature request!
```

```suggestion
        Thanks for taking the time to suggest this enhancement!
```
Did I miss this one last time? Sorry.